### PR TITLE
GUACAMOLE-201: Occasionally verify client responsiveness when intercepting streams.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: guacamole/src/main/webapp/app/rest/services/tunnelService.js [hunk 1]
```diff
@@ -215,6 +215,11 @@ angular.module('rest').factory('tunnelService', ['$injector',
             document.body.removeChild(iframe);
         };
 
+        // Acknowledge (and ignore) and received blobs
```
Comment (jmuehlner) [inline guacamole/src/main/webapp/app/rest/services/tunnelService.js]: Is this meant to say "any received blobs"?

Comment (mike-jumper) [inline guacamole/src/main/webapp/app/rest/services/tunnelService.js]: No. It's meant to say "Acknowledge (and ignore) any received blobs".

Comment (mike-jumper) [inline guacamole/src/main/webapp/app/rest/services/tunnelService.js]: ...but yes, you're right.

**Comment by jmuehlner** [comment]
Ok, this makes sense to me.

**Pull request merged**